src/lib.rs

```diff
@@ -573,5 +573,41 @@ pub mod pallet {
 			Votes::<T>::remove(supersig_index, call_index);
 			UsersVotes::<T>::remove_prefix((supersig_index, call_index), None);
 		}
+
+		pub fn add_to_supersig(supersig_idx: u128, members: Vec<(T::AccountId, Roles)>) -> u128 {
+			let mut number_added = 0;
+			for (member, role) in members {
+				if Self::members(supersig_idx, &member) == Roles::NotMember {
+					Members::<T>::insert(supersig_idx, member, role);
+					number_added += 1;
+				}
```

Comment:
What if I want to change the role of a member ?
Do we create another extrinsic, or just update it here ?
Second option seems easier for now

You will have to do an `else if current_role != role` and do the insert but not increment number_added